sofana/R3BSofFrsAnalysis.cxx

```diff
@@ -303,11 +304,12 @@ void R3BSofFrsAnalysis::Exec(Option_t* option)
         }
         if (isnan(tof) || tof < 0)
             continue;
-        beta[i] = fPathLength[i] / (tof + fTofOffset[i]); // ToFCalib
+        beta.push_back(fPathLength[i] / (tof + fTofOffset[i]));
     }
     // Velocity correlation conditions
-    if (fBetaCorr && beta[i_s2cave] > 0. && beta[i_s8cave] > 0. && TMath::Abs(beta[i_s2cave] - beta[i_s8cave] < 0.01))
-        beta[fNbTof] = beta[i_s2cave];
+    if (fBetaCorr && beta.at(i_s2cave) > 0. && beta.at(i_s8cave) > 0. &&
```

Comment:
Can it be like follows?:
`if (fBetaCorr && i_s2cave >= 0 && i_s8cave >= 0 && beta.at(i_s2cave) > 0. && beta.at(i_s8cave) > 0. &&`